### PR TITLE
docs: add man page

### LINKED ISSUE #273: Cross-platform documentation
at the moment as is, discordo has no extended thorough documentation about the keybindings, mouse actions, configuration. one figures them out on their own.

### PR DESCRIPTION
**Pull request by Matthieu-LAURENT39**
This aims to partially fix #273 by adding a man page

### CODE REVIEW

**Comment by sewnie** [comment]
Golang software isn't really made for man pages, but would benefit from a --help, or documentation.

File: man/discordo.1 [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+.SS Guild tree
+.TS
+allbox;
+l l 
+l l .
+\fB\fCAction\fR	\fB\fCKeybinding\fR
+Focus	Alt + g
+.TE
+
+.SS Message text
```
Comment (ayn2op) [inline man/discordo.1]: It is called "Messages Text". Similarly, "Guilds Tree" and "Message Input"

Comment (Matthieu-LAURENT39) [reply man/discordo.1]: Fixed it

**Comment by Matthieu-LAURENT39** [comment]
Is there anything else to fix before merging?

**Comment by ayn2op** [comment]
> Is there anything else to fix before merging?

LGTM! :rocket:

**Pull request merged**